integration/spark/app/src/test/java/io/openlineage/spark/agent/lifecycle/plan/SaveIntoDataSourceCommandVisitorTest.java

```diff
@@ -53,6 +58,16 @@ class SaveIntoDataSourceCommandVisitorTest {
 
   SparkOpenLineageExtensionVisitorWrapper wrapper =
       mock(SparkOpenLineageExtensionVisitorWrapper.class);
+
+  SparkListenerEvent event = mock(SparkListenerEvent.class);
+
+  StructType schema =
+      new StructType(
+          new StructField[] {
+            new StructField("key", IntegerType$.MODULE$, false, new Metadata(new HashMap<>())),
+            new StructField("value", StringType$.MODULE$, false, new Metadata(new HashMap<>()))
```

Comment:
Something slightly easier on the eyes is to use:

```suggestion
              new StructField("key", DataTypes.IntegerType, false, new Metadata(new HashMap<>())),
            new StructField("value", DataTypes.StringType, false, new Metadata(new HashMap<>()))
```

Reply:
Now that I think about it, there are several ways to do this:

```java
    StructType schema = new StructType()
        .add("key", DataTypes.IntegerType, false, Metadata.empty())
        .add("value", DataTypes.StringType, false, Metadata.empty());
    
    List<StructField> structFields = Arrays.asList(
        DataTypes.createStructField("key", DataTypes.IntegerType, false),
        DataTypes.createStructField("value", DataTypes.StringType, false)
    );
    StructType schema = DataTypes.createStructType(structFields);
```

Reply:
applied version 1